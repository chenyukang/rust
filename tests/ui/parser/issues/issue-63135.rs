// error-pattern: this file contains an unclosed delimiter
// error-pattern: aborting due to 2 previous errors
fn i(n{...,f #
