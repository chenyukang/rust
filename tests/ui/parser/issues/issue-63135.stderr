error: this file contains an unclosed delimiter
  --> $DIR/issue-63135.rs:3:16
   |
LL | fn i(n{...,f #
   |     - -        ^
   |     | |
   |     | unclosed delimiter
   |     unclosed delimiter

error: this file contains an unclosed delimiter
  --> $DIR/issue-63135.rs:3:16
   |
LL | fn i(n{...,f #
   |     - -        ^
   |     | |
   |     | unclosed delimiter
   |     unclosed delimiter

error: aborting due to 2 previous errors

